package.json

```diff
@@ -64,7 +64,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": ".husky/verify_branch_name && lint-staged && npm run test"
+      "pre-commit": ".husky/verify_branch_name && lint-staged && npm run test",
+      "commit-msg": ".husky/verify_commit_message && lint-staged"
```

Comment:
just to be sure - it's ok and safe to leave the "test" command in the pre-comit hook, no? 🤔